integration/redirects-test.ts

```diff
@@ -0,0 +1,174 @@
+import { test, expect } from "@playwright/test";
+
+import { createFixture, createAppFixture, js } from "./helpers/create-fixture";
+import type { Fixture, AppFixture } from "./helpers/create-fixture";
+import { PlaywrightFixture } from "./helpers/playwright-fixture";
+
+test.describe("redirects", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      files: {
+        "app/routes/action.jsx": js`
+          import { Outlet, useLoaderData } from "@remix-run/react";
+
+          if (typeof global.actionCount === "undefined") {
+            global.actionCount = 0;
+          }
+
+          export async function loader({ request }) {
+            return { count: ++global.actionCount };
```

Comment:
We use a count to confirm how many times this parent loader runs on the redirects